aries-site/src/examples/templates/forms/ShippingExample.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Main,
+  MaskedInput,
+  Select,
+  Text,
+  TextInput,
+} from 'grommet';
+
+import { FormContainer } from '.';
+import { emailMask, emailValidation, phoneMask, states } from './formHelpers';
+
+export const ShippingExample = () => {
+  const [formValues, setFormValues] = React.useState({});
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Heading level={3} margin="none">
+            Shipping
+          </Heading>
+          <Text>for your HPE products</Text>
+        </Header>
+        <Main
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
+            validate="blur"
+            value={formValues}
+            onChange={setFormValues}
+            onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+          >
+            <Box>
+              <Heading level={4} margin={{ bottom: 'small' }}>
```

Comment:
This heading level isn't bolded in the design -- but semantically I think it makes sense to be a header.

Reply:
Added the following to: https://github.com/hpe-design/design-system/issues/458
- Form section headings
  - Design calls for 'normal' weight. Should we be treating these section headings semantically as headings? If so, theme has headings bold vs. normal.